### DISCUSSION TITLE
Change Request: Remove the trim for the currency symbol

### DISCUSSION THREAD

**Discussion by Music4lity** [EssentialsX/Essentials#4111] [Ideas and Feature Suggestions] [answered]
Currently, EssX trims the string that is supplied for the currency symbol in config.yml down to the first character. My suggestion is that this trim is removed to allow for multiple characters in the currency symbol.

**Comment by JRoy** [comment, marked as answer]
This would break a multitude of things and is not what currency symbols are meant to do so this will not be added in 2.x.
